CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## 2.21.0-rc.0 / 2020-08-27
+
+This release is built with go1.15 which deprecates [X.509
```

Comment:
Nits:
- Comma before the "which".
- We usually write Go versions as "Go 1.15".